src/plugins/plot/src/configuration/YAxisModel.js

```diff
@@ -139,14 +139,14 @@ define([
             }, this);
         },
         trackSeries: function (series) {
-            this.listenTo(series, 'change:stats', function (seriesStats) {
+            this.listenTo(series, 'change:stats', (seriesStats => {
                 if (!seriesStats) {
                     this.resetStats();
                 } else {
                     this.updateStats(seriesStats);
                 }
-            }, this);
-            this.listenTo(series, 'change:yKey', function () {
+            }), this);
```

Comment:
Done. I assume it's also ok to remove the `this` arg at the end of the `listenTo`s where I'm using arrow functions?

Reply:
`listenTo` is defined in the helperFunctions file. I don't think you can remove `this` because it is an argument to the function call, but check the definition for `listenTo` to make sure.

Reply:
The pertinent line would seem to be eventHelpers.js line 40:

`_cb: context ? callback.bind(context) : callback`

Doesn't this imply that it should be ok to not pass `this` if the context of `this` is unchanged, as in arrow functions?

Reply:
Yeah that seems correct, but then also seems to warrant possibly changing the `listenTo` function. Which rule is this that enforces arrow functions anyways? I don't see anything wrong with the original code (even though I prefer arrows...).

Reply:
invalid-this is the rule that flags places where `this` is incorrectly used. It had objected to the use of `this` in some of the places where I tried to solve the problem by using arrow functions